include/solvers/simulation_parameters.h

```diff
@@ -31,8 +31,10 @@
 #include <solvers/initial_conditions.h>
 #include <solvers/source_terms.h>
 
+#include <dem/dem_solver_parameters.h>
+
 template <int dim>
-class SimulationParameters
+class SimulationParameters : public DEMSolverParameters<dim>
```

Comment:
I think this is not a good idea.
You can have a CFDSolverParameters, DEMSolverParameters and finally a CFD-DEMSolverParameters. It's not a good idea to have the base SimulationParameters have all the DEM properties. I don't think that's healthy.
I would make a whole new parameter class in include/femdem/ to agglomerate CFD-DEM parameters.  The coupling should remain in the Coupling library. The basic solver library should NOT depend on the DEM library. It does not work with the coupling architecture .

Reply:
done